exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/CreateMissingTablesAndColumnsTests.kt

```diff
@@ -366,7 +366,7 @@ class CreateMissingTablesAndColumnsTests : DatabaseTestsBase() {
                         actual.forEach { exec(it) }
                     } else {
                         SchemaUtils.drop(whiteSpaceTable)
-                        SchemaUtils.create(whiteSpaceTable)
+                        SchemaUtils.create(emptyTable)
```

Comment:
It was the error in the test, but it didn't occured before because database side defaults were not used.